tests/main.ml

```diff
@@ -228,7 +228,7 @@ module Check = struct
       ko ~error:(`Code 701) ~basename:"nested_comments_2.dk" [];
       ko ~error:(`Code 101) ~basename:"noninjectivity.dk" [];
       ko ~error:(`Code 507) ~basename:"nonleftlinear.dk" [Left_linear];
-      ko ~error:(`Code 701) ~basename:"nsteps1.dk" [];
+      ko ~error:(`Code 703) ~basename:"nsteps1.dk" [];
```

Comment:
Before it was raising a lexing error because "-1" was not a valid identifier. Now that '-' is accepted in identifiers, it raises a scoping error because of the new test added in menhir_parser.mly:
```
if n < 0 then raise Exit else ...
```